wandb/sdk/launch/agent/agent.py

```diff
@@ -118,6 +154,63 @@ def finish_job_id(self, job_id: Union[str, int]) -> None:
         if self._running == 0:
             self.update_status(AGENT_POLLING)
 
+    @staticmethod
+    def _parse_runner_config(config: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Parses a RunnersConfig block from the agent config. Returns a dictionary
+        of support runners in the form of:
+        {
+            kubernetes: {
+                namespace: "wandb"
+            },
+            sagemaker: {
+
+            },
+            local-process: {
+                labels: [ "gpu-pool" ]
+            },
+            local-container: {
+                labels: [ "gpu-pool"]
+            }
+        }
+        """
+        pass
+
+    def _filter_valid_queues(self) -> None:
+        """
+        Given an entity, return a list of run queues associated with that entity.
+        """
+        run_queues = self._api.get_run_queues_by_entity(self._entity)
+        valid_queues = []
+        for q in run_queues:
+            rc = q["resourceConfig"]
+            runner = rc["runner"]
+
+            if runner in self._configured_runners:
+                valid = runner_dispatch[runner]
```

Comment:
this should be `runner_dispatch[runner](self, rc)` or somethign like that, `valid` is a function right not, not the result